cmake/GenerateHelptags.cmake.in

```diff
@@ -1,10 +1,10 @@
 if(DEFINED ENV{DESTDIR})
   file(TO_CMAKE_PATH
-    "$ENV{DESTDIR}/${CMAKE_INSTALL_PREFIX}/share/nvim/runtime/doc"
+    "$ENV{DESTDIR}/@CMAKE_INSTALL_FULL_DATADIR@/nvim/runtime/doc"
```

Comment:
> I asked them if that was possible and they said no

Asked if what was possible?  We pass arguments in several places by defining variables on the command line.  Take a look at how we [run the unittests](https://github.com/neovim/neovim/blob/master/CMakeLists.txt#L334) as an example.

> so maybe they just didn't want me to do something kludgy

It's all kludgy, if you ask me. :-)  CMake can be great in many aspects, but it's lacking in many others.  The fact that we have to create these kinds of scripts everywhere being a prime example of that.